modules/ContactsNitroModule/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import type { ContactsModule } from './ContactsModule.nitro'
+import type { Contact } from './ContactsModule.nitro'
+import { NitroModules } from 'react-native-nitro-modules'
+
+export const ContactsNitroModule =
```

Comment:
Now, in here I think it would be reasonable to have a setup where this main module is just called `ExpensifyNativeUtils`, and composes more modularized pieces of code (like the contacts module) into multiple hybrid objects.

Then adding new kinds of native modules would be done here by:

- adding `MyNewNativeModule.nitro`
- running `nitrogen` to produce the native stubs
- implementing the native code
- registering the new HybridObject here

skipping all the boilerplate of adding a new podspec, build.gradle, etc...

Reply:
I’ve changed the approach a bit - added a draft empty module and set up a structure that makes it easy to add more modules like you described. The only thing I’d tweak is the name. Maybe call it `ExpensifyNitroUtils` or just `NitroUtils` instead of `ExpensifyNativeUtils` wdyt?